### PR TITLE
Migrate stellar-sdk beta.5, remove soroban-client

### PR DESCRIPTION
**Pull request by aristidesstaffieri**
This upgrades our js-stellar-sdk dependency to v11.0.0-beta.5.
This version brings in soroban-client and reogranizes the namespaces.

Changes include -
Removing all soroban-client references, replacing with stellar-sdk version.
Updating all stellar-sdk imports for new namespaces, no more importing entire lib.
Updates src/tests for API tweaks and namespace updates.

### CODE REVIEW

File: @shared/api/internal.ts [hunk 1]
```diff
@@ -313,20 +319,20 @@ export const getAccountBalances = async ({
...
-      allowHttp: networkUrl.startsWith("http://"),
-    },
-  });
-
   let balances: any = null;
   let isFunded = null;
   let subentryCount = 0;
 
   try {
+    const dataProvider = new DataProvider({
```
Comment (aristidesstaffieri) [inline @shared/api/internal.ts:327]: I found this edge case while testing coincidentally, if DataProvider throws when we instantiate it then we hit an uncaught error, so moved it into the existing try/catch.

**Review by piyalbasu — APPROVED** [review]

File: extension/src/popup/components/sendPayment/SendSettings/index.tsx [hunk 2]
```diff
@@ -98,9 +98,9 @@ export const SendSettings = ({
       );
 
       const params = [
-        new SorobanClient.Address(publicKey).toScVal(), // from
-        new SorobanClient.Address(destination).toScVal(), // to
-        new SorobanClient.XdrLargeInt("i128", parsedAmount.toNumber()).toI128(), // amount
+        new Address(publicKey).toScVal(), // from
+        new Address(destination).toScVal(), // to
+        new XdrLargeInt("i128", parsedAmount.toNumber()).toI128(), // amount
```
Comment (Shaptic) [inline extension/src/popup/components/sendPayment/SendSettings/index.tsx:103]: `new ScInt(parsedAmount).toI128()` is equivalent/shorter :shrug:

**Review by Shaptic — APPROVED** [review]
woo! :clap: can I ask how you felt overall with the new modularity?

File: extension/src/popup/components/sendPayment/SendSettings/index.tsx (hunk 2 above)
Comment (aristidesstaffieri) [reply extension/src/popup/components/sendPayment/SendSettings/index.tsx:103]: Thanks!

**Pull request merged**